app/upload/video/camera.tsx

```diff
@@ -0,0 +1,191 @@
+import { CameraView, CameraType, useCameraPermissions, FlashMode, useMicrophonePermissions } from 'expo-camera';
+import React, { useState, useRef, useEffect } from 'react';
+import { Text, View, Pressable, Platform} from 'react-native';
+import { supabase } from '../../../lib/supabase';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import { router } from 'expo-router';
+import { GestureDetector, Gesture } from 'react-native-gesture-handler';
+import { useMediaLibraryPermissions, launchImageLibraryAsync, MediaTypeOptions } from 'expo-image-picker';
+import { decode } from 'base64-arraybuffer';
+import { readAsStringAsync } from 'expo-file-system';
+import TopRow from '../../../components/upload/TopRow';
+import BottomRow from '../../../components/upload/video/BottomRow';
+import Camera from '../../../components/upload/video/Camera';
+
+export default function Page() {
+    const cameraRef = useRef<CameraView>(null);
```

Comment:
Do you think that you can separate these constants with some comments in order to make it more readable? Right now it is a little cluttered and hard to read.